fibbingnode/southbound/lsdb.py

```diff
@@ -466,7 +505,30 @@ def process_lsa(self):
                                  for part in lsa_info.split(SEP_GROUP) if part]
                     hdr = LSAHeader.parse(lsa_parts.pop(0))
                     lsa = LSA.parse(hdr, lsa_parts)
+                    new_seqnum = hdr.lsa_seqnum
+
                     log.debug('Parsed %s: %s', action, lsa)
+
+                    # For now, we just log lsa seq_num and ignore an LSA when
+                    # it has an older seq_num than the one in the database
+                    if hdr.lsa_type == '5':
```

Comment:
Could you expand the seqnum handling to all lsa types ? We basically to keep our view of the LSDB totally in sync with the network, so applying the seqnum handling to all LSA types makes sense.